documentation/src/main/asciidoc/faqs/faqs.asciidoc

```diff
@@ -16,56 +16,56 @@ any questions you may have on the link:http://www.infinispan.org/community[Infin
 === What is Infinispan?
 
 Infinispan is an open source data grid platform.
-It exposes a link:$$http://jcp.org/en/jsr/detail?id=107$$[JSR-107] compatible link:$$http://docs.jboss.org/infinispan/6.0/apidocs/org/infinispan/Cache.html$$[`Cache`] interface (which in turn extends `java.util.Map`) in which you can store objects.
+It exposes a link:{javadocroot}/org/infinispan/Cache.html$$[`Cache`] interface (which in turn extends `java.util.Map`) in which you can store objects.
```

Comment:
We don't actually need to use any notation unless special characters get escaped by asciidoc